fix_bank_account_status.rb

```
@@ -0,0 +1,72 @@
+#!/usr/bin/env ruby
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)